### PR TITLE
First attempt at adding centos7

### PR DESCRIPTION
**Pull request by jcftang**
Add the bits to build a centos7 set of binaries

I'm not entire sure how these release scripts run, but here's a first pass at this.

### CODE REVIEW

**Comment by purpleidea** [comment]
@jcftang Thanks for the patch! It looks good, except you'll need to change the commit message to pass our tests. Something like `misc: Blah blah` with that prefix will work.

While you're at it, one small nit pick: please re-order CentOS-7 so it comes right after Fedora-29. I think it's more logical that way for my crazyness of "modern first". Thank you!

**Comment by purpleidea** [comment]
You missed at least one entry (in the .PHONY at the top) -- just search for "29" to see where it's missing, and please make the commit message more meaningful. No need to say "first attempt". You say what it is about. We hide our mistakes in rebases so even if it took ten attempts to get this right (it won't) it looks clean when merged =D

Thanks again!

**Comment by jcftang** [comment]
Yea sorry about missing a few bits, I've fixed that. Trying to keep this PR to one commit so its cleaner for you guys

File: Makefile [hunk 1]
```diff
@@ -438,6 +447,14 @@ $(PKG_FEDORA-29): releases/$(VERSION)/$(TOKEN_FEDORA-29)/changelog
 	@title='$(@D)' ; distro=$${title#'releases/$(VERSION)/'} ; echo "Building: $${distro} package..."
 	@title='$(@D)' ; distro=$${title#'releases/$(VERSION)/'} ; ./misc/fpm-pack.sh $${distro} $(VERSION) "$(FILE_FEDORA-29)" libvirt-devel augeas-devel
 
+releases/$(VERSION)/$(TOKEN_CENTOS-7)/changelog: $(PROGRAM) releases/$(VERSION)/.mkdir
+	@title='$(@D)' ; distro=$${title#'releases/$(VERSION)/'} ; echo "Generating: $${distro} changelog..."
+	@title='$(@D)' ; distro=$${title#'releases/$(VERSION)/'} ; ./misc/make-rpm-changelog.sh "$${distro}" $(VERSION)
+
+$(PKG_CENTOS-7): releases/$(VERSION)/$(TOKEN_CENTOS-7)/changelog
+	@title='$(@D)' ; distro=$${title#'releases/$(VERSION)/'} ; echo "Building: $${distro} package..."
+	@title='$(@D)' ; distro=$${title#'releases/$(VERSION)/'} ; ./misc/fpm-pack.sh $${distro} $(VERSION) "$(FILE_FEDORA-30)" libvirt-devel augeas-devel
```
Comment (purpleidea) [inline Makefile]: copy-pasta!

Comment (jcftang) [reply Makefile]: yea is this correct to do?

**Comment by jcftang** [comment]
Ah I see what's wrong

**Comment by purpleidea** [comment]
LGTM, thanks!

**Comment by purpleidea** [comment]
@jcftang I rebased your patch to master and merged it.

https://github.com/purpleidea/mgmt/commit/d64f9f5401908eb7584566e988a9243ae6d5cc71

While the patch itself was good, centos-7 needed a bunch of tweaking, so I added that in a separate commit.

https://github.com/purpleidea/mgmt/commit/d57f7aa03f419dfc7f9f47aeb06fdf210f95afaf

Thanks again! I hope we can convince you to dive in to something new?

**Comment by jcftang** [comment]
Sorry for missing those tweaks, I didn't have much time to do a test run on those changes, next time!

**Pull request closed without merging**